tests/cases/dom/traverse/firstChild.js

```diff
@@ -0,0 +1,51 @@
+import {firstChild, findOne} from "../../../../dom/traverse";
+import QUnit from "qunit";
+
+QUnit.module("dom/traverse/firstChild()",
+    {
+        beforeEach: () =>
+        {
+            findOne("#qunit-fixture").innerHTML = `
+                <div class="element1"></div>
+                <div class="element2"></div>
+                Test123
+            `;
+        },
+    }
+);
+
+
+QUnit.test(
+    "firstChild() without selector",
+    (assert) =>
+    {
+        const fixture = findOne("#qunit-fixture");
+
+        const result = firstChild(fixture);
+        assert.ok(result.classList.contains("element1"), "contains .element1");
+    }
+);
+
+
+QUnit.test(
+    "firstChild() with selector matching two elements",
+    (assert) =>
+    {
+        const fixture = findOne("#qunit-fixture");
+
+        const result = firstChild(fixture, "div");
+        assert.ok(result.classList.contains("element1"), "the match has the correct class");
+    }
+);
+
+
+QUnit.test(
+    "firstChild() with selector",
+    (assert) =>
+    {
+        const fixture = findOne("#qunit-fixture");
+
+        const result = firstChild(fixture, ".element2");
+        assert.ok(result.classList.contains("element2"), "the match has the correct class");
+    }
```

Comment:
Can you please add a test, that tests the path that the selector matches nothing?

Reply:
I've went ahead and added the same test for `children`, since it was missing there as well.